src/GoTrueApi.ts

```diff
@@ -32,6 +32,46 @@ export default class GoTrueApi {
     this.cookieOptions = { ...COOKIE_OPTIONS, ...cookieOptions }
   }
 
+  /**
+   * Creates a new user.
+   *
+   * This function should only be called on a server. Never expose your `service_role` key in the browser.
+   *
+   * @param attributes The data you want to create the user with.
+   * @param jwt A valid JWT. Must be a full-access API key (e.g. service_role key).
+   */
+  async createUser(
+    attributes: UserAttributes
+  ): Promise<{ data: null; error: ApiError } | { data: User; error: null }> {
+    try {
+      const data: any = await post(`${this.url}/admin/users`, attributes, {
+        headers: this.headers,
+      })
+      return { data, error: null }
+    } catch (e) {
+      return { data: null, error: e as ApiError }
+    }
+  }
+
+  /**
+   * Get a list of users.
+   *
+   * This function should only be called on a server. Never expose your `service_role` key in the browser.
+   *
+   * @param attributes The data you want to create the user with.
+   * @param jwt A valid JWT. Must be a full-access API key (e.g. service_role key).
+   */
+  async listUsers(): Promise<{ data: null; error: ApiError } | { data: User[]; error: null }> {
+    try {
+      const data: any = await get(`${this.url}/admin/users`, {
```

Comment:
Thanks for adding this to the API! I think accepting a filter parameter in the `listUsers` method is also going to be very helpful. (Related usecase: https://github.com/supabase/gotrue/issues/180).

Reply:
I'll merge this for now and then we can add this in the future 👍